pub-matic-htb.js

```diff
@@ -370,21 +516,25 @@ function PubMaticHtb(configs) {
 
         /* ---------- Please fill out this partner profile according to your module ------------*/
         __profile = {
-            partnerId: 'PubMaticHtb', // PartnerName
-            namespace: 'PubMaticHtb', // Should be same as partnerName
+            partnerId: Constants.PUBMATIC_PARTNER_ID, // PartnerName
+            namespace: Constants.PUBMATIC_PARTNER_ID, // Should be same as partnerName
             statsId: 'PUBM', // Unique partner identifier
-            version: '2.0.0',
-            targetingType: 'slot',
+            version: '2.2.0',
```

Comment:
What was the reason for this change?

Reply:
since the existing pubmatic adapter in index used 2.1.0, we used 2.2.0 for this one. Should it be changed back to 2.1.0?